k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add-contexts.yml

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: aries-add-contexts
+  labels:
+    component: aries-framework
+    group: services
+spec:
+  template:
+    spec:
+      volumes:
+        - name: script
+          configMap:
+            name: aries-add-contexts-config
+      restartPolicy: Never
+      initContainers:
+        - name: adapters-ready
+          image: fedora
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/sh"]
+          args:
+            - "-c"
+            - |
+              ENDPOINTS=(adapter-issuer adapter-rp);
+              for endpoint in "${ENDPOINTS[@]}";
+              do while [[ "$(curl -o /dev/null -s -w '%{http_code}' --insecure --connect-timeout 5 https://${endpoint}.||DOMAIN||/healthcheck)" != "200" ]];
+              do echo "waiting for ${endpoint} endpoint";
+              sleep 5;
+              done;
+              done;
+        - name: demo-components-ready
+          image: fedora
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/sh"]
+          args:
+            - "-c"
+            - |
+              ENDPOINTS=(demo-issuer demo-rp);
+              for endpoint in "${ENDPOINTS[@]}";
+              do while [[ "$(curl -o /dev/null -s -w '%{http_code}' --insecure --connect-timeout 5 https://${endpoint}.||DOMAIN||/healthcheck)" != "200" ]];
+              do echo "waiting for ${endpoint} endpoint";
+              sleep 5;
+              done;
+              done;
+        - name: vcs-components-ready
```

Comment:
might want to move this to become the second entry before the demo-issuer/rp

Reply:
removed demo-issuer/rp as no need to add contexts for them thru API